dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -252,6 +252,114 @@ html, body {
   }
 }
 
+/* Styling that keeps the small footer consistent with other page elements */
+$codeApp-left-margin: 25px;
+$small-footer-horizontal-margin: 15px;
+$small-footer-standard-width: 400px;
+
+/* Shared styles so flyout looks like footer */
+.small-footer, #copyright-flyout, #more-menu {
+  background-color: #ddd; /* Matches blockly flyout */
```

Comment:
I think that matching the coloring to blockly-flyout-color is done a a stylistic choice, but it could easily be colored differently and probably still look okay. Because of that, I prefer a comment to any tighter binding between the two.